src/modules/event/infra/typeorm/repositories/EventUserRepository.ts

```diff
@@ -0,0 +1,52 @@
+import IEventUserDTO from '@modules/event/dtos/IEventUserDTO';
+import IEventUserRepository from '@modules/event/repositories/IEventUserRepository';
+import { getRepository, Repository } from 'typeorm';
+import EventUser from '../entities/EventUser';
+
+export default class EventUserRepository implements IEventUserRepository {
+  private ormRepository: Repository<EventUser>;
+
+  constructor() {
+    this.ormRepository = getRepository(EventUser);
+  }
+
+  public async findAllEventUser(user_id?: string): Promise<EventUser[]> {
```

Comment:
the ideal would be to separate this function in two, one responsible only for the Index and the other for showing the user's events. This function currently breaks the Single-responsibility principle.